### PR TITLE
Added try-catches to CookieVisitor.cpp to avoid crashes on invalid va…

### PR DESCRIPTION
**Pull request by YaiEf**
…lues for DateTime.

This is the simplest possible fix. It ignores cases where the user might want to know what the invalid dates were. If that is important, I would suggest creating a new class (CefDateTime?) that directly corresponds to the values in CefCookie, and then create DateTime on demand in Cookie.cs.

### CODE REVIEW

**Comment by amaitland** [comment]
> I would suggest creating a new class (CefDateTime?) that directly corresponds to the values in CefCookie, and then create DateTime on demand in Cookie.cs.

If you feel so inclined you can :+1:

**Comment by amaitland** [comment]
@YaiEf In general, spaces instead of tabs, you can review your `PR` above, you'll see the formatting is out.

File: CefSharp.Core/Internals/CookieVisitor.cpp [hunk 1]
```diff
@@ -21,39 +21,54 @@ namespace CefSharp
...
-                    cefCookie.expires.year,
-                    cefCookie.expires.month,
-                    cefCookie.expires.day_of_month,
-                    cefCookie.expires.hour,
-                    cefCookie.expires.minute,
-                    cefCookie.expires.second,
-                    cefCookie.expires.millisecond
-                );
-            }
+			try {
```
Comment (amaitland) [inline CefSharp.Core/Internals/CookieVisitor.cpp]: Braces on new lines please.

File: CefSharp.Core/Internals/CookieVisitor.cpp [hunk 2]
```diff
@@ -21,39 +21,54 @@ namespace CefSharp
...
+						cefCookie.expires.day_of_month,
+						cefCookie.expires.hour,
+						cefCookie.expires.minute,
+						cefCookie.expires.second,
+						cefCookie.expires.millisecond
+						);
+				}
+			}
+			catch (Exception^ ex){
+				// ignore invalid date as we have no way to represent it
```
Comment (amaitland) [inline CefSharp.Core/Internals/CookieVisitor.cpp]: Could probably set to `MinValue` to represent an invalid `DateTime` for now.

**Comment by YaiEf** [comment]
I have updated it so it hopefully conforms now :)

**Comment by amaitland** [comment]
Thanks :+1:

**Comment by zhyzhy782** [comment]
Yo! 

I've got that nice stuff  for  you, it is  awesome. Just take a look at it <http://favor.crowandraven.org/e7yv/11>

All the best, zhyzhy782

**Pull request merged**